disnake/audit_logs.py

```diff
@@ -180,6 +185,40 @@ def _transform(entry: AuditLogEntry, data: Optional[str]) -> Optional[Asset]:
     return _transform
 
 
+def _transform_tag(entry: AuditLogEntry, data: Optional[ForumTagPayload]) -> Optional[ForumTag]:
+    if data is None:
+        return None
+    from .threads import ForumTag  # cyclic import
```

Comment:
perhaps tags can be in their own file? not sure.

Reply:
The `.threads` import wasn't actually causing issues (I might have done that before), so I moved it in https://github.com/DisnakeDev/disnake/pull/726/commits/f341b1afd9f4bbde2ad503d15e9b225e8c9ffa4b. Moving the `.channel` import is still problematic due to iterators creating lots of dependencies between modules, but that's fine for now.